src/Extracting/Strategies/Strategy.php

```diff
@@ -24,6 +24,16 @@ public function __construct(DocumentationConfig $config)
         $this->config = $config;
     }
 
+    /**
+     * Returns a copy of the documentation config
+     *
+     * @return DocumentationConfig
+     */
+    public function getConfig()
```

Comment:
Curious, why did you add this method?

Also, it says it returns a copy of the config, but that isn't correct. It returns the same object.

Reply:
Because `DatabaseTransactionsHelper` is a trait that could be applied anywhere, and not exclusively to a child class of the abstract `Strategy`, the trait can't guarantee there is access to a instance property `$config`. Personally I think this is potentially something the language could add, however the correct approach at the trait level, would be to add the requirement for a method that would return an instance of the Config. Such method does not currently exist, so hence the new method.

Whilst I could have used in in-built `config` method from Laravel/Lumen, I felt you might prefer it if I stick to instances of the `DocumentationConfig`.

Reply:
Fixed the description. Hopefully it's clearer.